packages/js/product-editor/src/style.scss

```diff
@@ -6,7 +6,9 @@
 @import 'components/header/style.scss';
 @import 'components/images/editor.scss';
 @import 'components/block-editor/style.scss';
+@import 'components/modal-editor/style.scss';
```

Comment:
This style sheet doesn't exist, which breaks the build

Reply:
Whoops!  Must be some funkiness after the rebase/rename.  Removed.